### DISCUSSION TITLE
Issues accessing `MjSpec.visual.headlight`

### DISCUSSION THREAD

**Discussion by AaronYoung5** [google-deepmind/mujoco#2289] [Asking for Help]
### Intro

Hi!

I'm trying to update my codebase to use MjSpec and am running into issues when trying to dynamically set the headlight to inactive. I get the same error for `visual.rgba`.

```bash
$ python -c 'import mujoco; print(mujoco.MjSpec().visual.headlight)'

TypeError: Unregistered type : mjVisual_::{unnamed type#3}

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: Unable to convert function return value to a Python type! The signature was
        (arg0: mujoco._specs.MjVisual) -> mjVisual_::{unnamed type#3}
```

Additionally, it seems like `MjSpec.visual.global` wasn't renamed in the python bindings like for `MjModel.global_`.

```bash
$ python -c 'import mujoco; print(mujoco.MjSpec().visual.global)'
  File "<string>", line 1
    import mujoco; print(mujoco.MjSpec().visual.global)
                                                ^^^^^^
SyntaxError: invalid syntax

# Workaround
$ python -c 'import mujoco; print(getattr(mujoco.MjSpec().visual, "global"))'
...
```

### My setup

I've tested with `mujoco==3.2.6`, as well as building from source. I'm using Python 3.11.9.

### My question

Am I doing something wrong in either case? It doesn't look like the spec_tests for either case. I can file a bug report, if necessary. Also happy to help if someone can point me in the right direction.

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by yuvaltassa** [comment]
@quagla can you take a look?

**Reply by AaronYoung5** [reply]
Ended up trying myself this morning, I can make a PR if this is the right way to go about it. Works as expected for me now.

https://github.com/AaronYoung5/mujoco/commit/ba25a690d5f7ec2b890741359d7a32fe504d5350

**Reply by yuvaltassa** [reply]
This looks good. A PR would be appreciated, thanks.

**Reply by AaronYoung5** [reply]
https://github.com/google-deepmind/mujoco/pull/2291
